css/main.css

```diff
@@ -1,3 +1,222 @@
-/* Add your CSS here */
+/* 
+Fonts
+Headings: Poppins - Semi-bold 600
+https://fonts.google.com/specimen/Poppins
+Body: Quicksand - Light 300
+https://fonts.google.com/specimen/Quicksand
+
+Colors
+Dark beige: #E6DACD
+Light beige: #F4ECE6
+Royal blue: #0150FD
+Black: #000
+White: #FFF */
 
+/* Add your CSS here */
 /* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --dark-beige: #e6dacd;
+  --light-beige: #f4ece6;
+  --royal-blue: #0150fd;
+  --black: #000;
+  --white: #fff;
+}
+
+body {
+  width: 320px;
+  margin: 0;
+  padding: 0;
+  font-family: "Quicksand", sans-serif;
+}
+/*Headings: font-family: "Poppins", sans-serif; */
+.content {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "header"
+    "intro"
+    "main"
+    "footer";
+}
+
+/* header style */
+header {
+  padding-left: 6%;
+  padding-top: 4%;
+  margin-bottom: 36%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "par par par ham"
+    ". . .  ham";
+}
+.hamburger-menu {
+  grid-area: ham;
+  padding-top: 18%;
+  height: 20px;
+  padding-left: 20px;
+}
+
+main {
+  position: relative;
+  padding-top: 10px;
+}
+.profile-img {
+  display: block;
+  border-radius: 50%;
+  width: 44%;
+  margin: 15px auto 18px auto;
+}
+
+header > p:nth-child(1) {
+  margin: 0;
+  display: grid;
```

Comment:
This is the same as last week, I'm guessing you wrote this before you saw my feedback, but just remember you don't need to use display: grid on the child, only the parent